core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -126,9 +126,10 @@ var PreferencesPage = function() {
   };
 
   this.setUserInterests = async function(interests) {
-    await userInterestsInput.click();
+    await action.click('User Interest Input', userInterestsInput);
     for (var i = 0; i < interests.length; i++) {
-      await userInterestsInput.sendKeys(interests[i], protractor.Key.RETURN);
+      await action.sendKeys(
+        'User Interest Input', userInterestsInput, interests[i]);
```

Comment:
Hm, does this work? It doesn't seem equivalent to the previous code where it simulates hitting the enter key.

Reply:
Have you tried running this suite locally?

Reply:
> Hm, does this work? It doesn't seem equivalent to the previous code where it simulates hitting the enter key.

Ok I will get to it. I was thinking the same at the first instance but was confused with number of arguments to be passed into that. Will roll back to its original form.